src/test/java/org/prebid/server/bidder/ttx/TtxBidderTest.java

```diff
@@ -129,6 +136,120 @@ public void makeHttpRequestsShouldChangeOnlyFirstImpExt() {
+    }
+
+    @Test
+    public void makeHttpRequestsShouldUpdateNotPresentPlacement() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(asList(
+                        givenImp(impBuilder -> impBuilder
+                                .video(Video.builder()
+                                        .w(23)
+                                        .h(23)
+                                        .mimes(singletonList("mime"))
+                                        .protocols(singletonList(23))
+                                        .playbackmethod(singletonList(27))
+                                        .build())
+                                .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpTtx.of("11", null, "3")))))
+                ))
+                .build();
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = ttxBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue())
+                .extracting(HttpRequest::getPayload)
+                .flatExtracting(BidRequest::getImp)
+                .extracting(Imp::getVideo)
+                .containsExactly(Video.builder()
+                        .placement(2)
+                        .w(23)
+                        .h(23)
+                        .mimes(singletonList("mime"))
+                        .protocols(singletonList(23))
+                        .playbackmethod(singletonList(27))
+                        .build());
+    }
+
+    @Test
+    public void makeHttpRequestsShouldUpdatePlacementAndStartDelayIfProdIsInstream() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(asList(
+                        givenImp(impBuilder -> impBuilder
+                                .video(Video.builder()
+                                        .w(23)
+                                        .h(23)
+                                        .mimes(singletonList("mime"))
+                                        .protocols(singletonList(23))
+                                        .playbackmethod(singletonList(27))
+                                        .build())
+                                .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpTtx.of("11", null, "instream")))))
+                ))
+                .build();
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = ttxBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue())
+                .extracting(HttpRequest::getPayload)
+                .flatExtracting(BidRequest::getImp)
+                .extracting(Imp::getVideo)
+                .containsExactly(Video.builder()
+                        .placement(1)
+                        .startdelay(0)
```

Comment:
Check only for this two values. (or extract Video builder and update it in 'then' section to create expected one)